azavea/raster-foundry
# Feature/rename models to tools

## Overview

Rename Models to Tools

### Checklist


### Notes

- [X] Based on #741 - will rebase onto develop once it gets merged, so only the second commit is relevant

I renamed the ingests packages where I thought it made sense, but I'm not really sure how to test them

## Testing Instructions

* Run migrations
* Verify that the database tables have no references to models
  * In the vagrant machine, run `./scripts/psql`
  * type `\d` to verify that `model_tags` and `model_categories` no longer exist
  * type '\d tool_tags` to verify that there are no references to a models in the fields or constraints
  * type `\d tool_categories` to verify that there are no references to models in the fields or constraints
